tools/install/Makefile.skel

```diff
@@ -22,23 +21,21 @@ FORTC = "gnu"
 
 # set the dependencies paths
 # N.B.: use the FULL PATHS, not the relative paths
-CVODELIB     = cvode/lib
+CVODELIBDIR     = cvode/lib
 OPENLIBMDIR  = openlibm-0.8.1
 FRUITDIR     = fruit_3.4.3
 
 # default location of the chemical mechanism shared library (mechanism.so)
 # use the second argument of the build script to override $SHAREDLIBDIR
 SHAREDLIBDIR = model/configuration
 
-
 # =========================================================================== #
 # DO NOT MODIFY BELOW THIS LINE
 # =========================================================================== #
 
-
 .SUFFIXES:
 .SUFFIXES: .f90 .o
-.PHONY: all test
+.PHONY: all indenttest styletest unittests oldtests modeltests alltests clean
```

Comment:
This is confusing to me: should all the rules that do not generate a file listed here? If os, why is `all` which generates the executable?

Reply:
I'm really not sure about this, I'm sorry

Reply:
Make uses timestamps on files to decide whether to run a a target or skip it - no point in rerunning a target if none of its dependencies have changed. Make typically assumes a given rule creates a file with the same name.

In the case where this is not true (for example, `all`, `test` or indeed any of the `*tests` above), it will always run the target rule, because there's no file there with a timestamp to check.

However, `.PHONY` is provided as a safeguard in case someone inadvertently creates a file with that name - say someone created a file called `modeltests`. `.PHONY` tells make to ignore the presence of the file, and run the rule anyway. So it's just to safeguard against the scenario where someone creates such a file, runs the rule, make checks the timestamp and dependencies, doesn't run the rule, but just reports success anyway.

TL;DR: it's not vital either way, but best to add all of those to `.PHONY` as you've done above.

Reply:
Thanks for clarifying this. It makes sense. 
So if I now move the `*tests` tules to the new `makefile.tests` I need to split the .PHONY as well? it's not like a variable that gets overwritten when make executes the `include` instruction?

Reply:
https://stackoverflow.com/questions/59294990/how-should-you-use-phony-in-included-makefiles is quite helpful. Essentially, you can do either, but it might make most sense to split it.

Reply:
Thanks! I have been scouring stackoverflow for that, but somehow missed that question. Now it all clear.